lib/modules/datasource/rpm/readme.md

```diff
@@ -0,0 +1,2 @@
+This datasource returns releases of the RPM packages.
+It assumes the RPM repository is following the RPM standard and by default it has a repomd.xml in the directory provided by user in the `registryUrl`.
```

Comment:
I'd expand on this more if you can. Is there an external link you can reference for `repomd.xml`? And can you provide an example of how to set up this datasource in `renovate.json`? Look at the `deb` datasource readme as a guide.

Reply:
Updated with more context. Thanks